docs/source/getting_started/cpu-installation.rst

```diff
@@ -20,7 +20,7 @@ Requirements
 
 * OS: Linux
 * Compiler: gcc/g++>=12.3.0 (optional, recommended)
-* Instruction set architecture (ISA) requirement: AVX512 is required.
+* Instruction set architecture (ISA) requirement: for x86, AVX2 is required; for PowerPC, Power9+ is required.
```

Comment:
Since the AVX2 and PowerPC backends don't have active testing and this documentation assumes you are building on a machine with AVX512, I'm a little hesitant to update in this doc. Maybe if you call out specifically where you can get directions for AVX2 and PowerPC, and that this doc is assuming AVX512, that would be more clear.

Reply:
Thanks @mgoin for the review.

The origin description `AVX512 is required` is quite misleading, which means vLLM requires AVX512 ISA to run.
It would limit the usage of vLLM if people wrongly get the msg that vLLM only supports AVX512.

Actually, we've built and run vLLM on AVX2-only machines.
So I would suggest removing `AVX512 is required` here.

However, if you're against adding the PowerPC CPUs here, I'm fine.
This is because we don't have PowerPC CPUs and don't test it at all.

Thanks.

Reply:
> Since the AVX2 and PowerPC backends don't have active testing and this documentation assumes you are building on a machine with AVX512, I'm a little hesitant to update in this doc. Maybe if you call out specifically where you can get directions for AVX2 and PowerPC, and that this doc is assuming AVX512, that would be more clear.

Hi @mgoin , I kept AVX512 in the doc.
But make it to be (optional, recommended) as the description of the compiler.
What do you think?
Thanks.